dist/rockcraft-pack-action/index.js

```diff
@@ -4286,20 +4287,22 @@ class RockcraftBuilder {
     }
 }
 
-;// CONCATENATED MODULE: ./src/rockcraft-pack-action.ts
+;// CONCATENATED MODULE: ./lib/rockcraft-pack-action.js
 // -*- mode: javascript; js-indent-level: 2 -*-
 
 
 async function run() {
     try {
         const projectRoot = core.getInput('path');
         core.info(`Building ROCK in "${projectRoot}"...`);
-        const rockcraftChannel = core.getInput('rockcraft-channel') || 'edge';
+        const rockcraftChannel = core.getInput('rockcraft-channel') || 'stable';
```

Comment:
good catch, maybe make this a separate PR so we roll it out faster

Reply:
well, better not cause there are people relying on this behavior to get the latest features from edge. changing this without a way for them to still be able to refer to `latest/edge` would block them